samigo/samigo-app/src/webapp/jsf/author/publishedSettings.jsp

```diff
@@ -616,6 +616,9 @@
             </li>
           </ul>
           <h:outputLabel id="selTimeAssessHelpBlock" styleClass="help-block info-text small" value="#{assessmentSettingsMessages.assessment_timed_info}" />
+          <h:panelGroup rendered="#{publishedSettings.assessment.hasMultipleTimers() > 1}" layout="block" id="multipleTimersInfo" styleClass="sak-banner-warn">
+            <h:outputText value="#{assessmentSettingsMessages.multiple_timers_detected}" />
+          </h:panelGroup>
```

Comment:
Same, move it to the confirmation screen.

Reply:
Duplicated in confirmation publish screen. Message will be shown in both screens